src/pages/AppList/AppListComponent.tsx

```diff
@@ -143,6 +144,15 @@ class AppListComponent extends FilterComponent.Component<AppListComponentProps,
           ]}
         />
       </VirtualList>
+    ) : (
+      <EmptyState variant={EmptyStateVariant.full}>
+        <Title headingLevel="h5" size="lg">
+          No Applications found
+        </Title>
+        <EmptyStateBody>
+          No applications in namespaces : {this.props.activeNamespaces.map(ns => ns.name).join(', ')}
```

Comment:
Nitpick:
The space between `namespaces` and `:` is a bit annoying, isn't it?
Besides that, what do you think to change the copy depending on the namespace count: `No applications in default namespace` or `No applications in namespaces: blah, blah, blah`